WordPress/Classes/ViewRelated/Activity/BaseActivityListViewController.swift

```diff
@@ -394,17 +394,17 @@ extension BaseActivityListViewController: ActivityPresenter {
 
         guard FeatureFlag.jetpackBackupAndRestore.enabled else {
             let title = NSLocalizedString("Rewind Site",
-                                          comment: "Title displayed in the Rewind Site alert, should match Calypso")
+                                          comment: "Title displayed in the Restore Site alert, should match Calypso")
             let rewindDate = viewModel.mediumDateFormatterWithTime.string(from: activity.published)
-            let messageFormat = NSLocalizedString("Are you sure you want to rewind your site back to %@?\nThis will remove all content and options created or changed since then.",
+            let messageFormat = NSLocalizedString("Are you sure you want to restore your site back to %@?\nAnything you changed since then will be lost..",
```

Comment:
Two points in the end:

```suggestion
            let messageFormat = NSLocalizedString("Are you sure you want to restore your site back to %@?\nAnything you changed since then will be lost.",
```